test/signhash.ts

```diff
@@ -142,4 +142,66 @@ contract('SignHash', accounts => {
       });
     });
   });
+
+  describe('#removeProof', () => {
+    it('should remove proof method', async () => {
+      const signer = accounts[0];
```

Comment:
Most of the tests use just `accounts[0]` as a signer, it could be defined on a higher level to avoid that duplication. Also, I know this should not matter in our case, but deploying the contract from some other account than the signing one might be a good idea.

Reply:
Fixed in e337f0f